src/components/GlobalMap/ShipDetailsMap/ShipDetailsMap.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
I think that we can FiltersVessel component move in GlobalMap folder (like ShipDetailsMap) ?